src/mpl/src/gavl/mpl_gavl.c

```diff
@@ -164,82 +195,66 @@ static int gavl_intersect_cmp_func(uintptr_t ustart, uintptr_t len, gavl_tree_no
 int MPL_gavl_tree_create(void (*free_fn) (void *), MPL_gavl_tree_t * gavl_tree)
 {
     int mpl_err = MPL_SUCCESS;
-    gavl_tree_s *gavl_tree_iptr;
+    gavl_tree_s *tree_ptr;
 
-    gavl_tree_iptr = (gavl_tree_s *) MPL_malloc(sizeof(gavl_tree_s), MPL_MEM_OTHER);
-    if (gavl_tree_iptr == NULL) {
+    tree_ptr = (gavl_tree_s *) MPL_malloc(sizeof(gavl_tree_s), MPL_MEM_OTHER);
+    if (tree_ptr == NULL) {
         mpl_err = MPL_ERR_SHM_NOMEM;
         goto fn_fail;
```

Comment:
The errcode seems wrong. It is not `SHM` related.  You can define new code such as `MPL_ERR_NOMEM`

Reply:
Search `**nomem`, it is already defined and used in many other places.